arcgis-ios-sdk-samples/Maps/Manage bookmarks/BookmarksViewController.swift

```diff
@@ -112,26 +112,26 @@ class BookmarksViewController: UIViewController, UIAlertViewDelegate, UIAdaptive
         self.alertController.addAction(doneAction)
         
         //present alert controller
-        self.presentViewController(self.alertController, animated: true, completion: nil)
+        self.present(self.alertController, animated: true, completion: nil)
     }
     
-    private func addBookmark(name:String) {
+    fileprivate func addBookmark(_ name:String) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage